README.md

```diff
@@ -1,60 +1,63 @@
-* You should have a CPU that's running at 1GHz or better.
+To run SuperTuxKart, make sure that your computer's specifications are equal or higher than the following specifications:
+
+* A graphics card capable of 3D rendering - NVIDIA GeForce 8 series and newer (GeForce 8100 or newer), AMD/ATI Radeon HD 4000 series and newer, Intel HD Graphics 3000 and newer. OpenGL >= 3.3
+* You should have a CPU that's running at 1 GHz or faster. 
 * You'll need at least 512 MB of free VRAM (video memory).
-* Disk space: 400MB
-* Ideally, you want a joystick with at least 6 buttons.
+* Minimum disk space: 25 GB 
+* Ideally, you'll want a joystick with at least 6 buttons.
 
 ## License
-This software is released under the GNU General Public License (GPL) which can be found in the file [`COPYING`](/COPYING) in the same directory as this file. Information about the licenses for artwork are contained in `data/licenses`.
+The software is released under the GNU General Public License (GPL) which can be found in the file [`COPYING`](/COPYING) in the same directory as this file. Information about the licenses for the artwork is contained in `data/licenses`.
 
 ## 3D coordinates
-A reminder for those looking at the code and 3d models:
+A reminder for those who are looking at the code and 3D models:
 
-STK    : X right, Y up,       Z forwards
+SuperTuxKart: X right, Y up, Z forwards
 
 Blender: X right, Y forwards, Z up
 
-The exporters perform the needed transform, so in Blender you just work with XY plane as ground, and things will appear fine in STK (using XZ as ground in the code, obviously).
+The export utilities  perform the needed transformation, so in Blender you just work with the XY plane as ground, and things will appear fine in STK (using XZ as ground in the code, obviously).
 
 ## Building from source
 
-First, you need both the code and the assets (See <https://supertuxkart.net/Source_control> for more information):
+In order to build SuperTuxKart from source, you'll need both the code and the assets (See <https://supertuxkart.net/Source_control> for more information):
 
-```
-git clone https://github.com/supertuxkart/stk-code
-svn checkout https://svn.code.sf.net/p/supertuxkart/code/stk-assets stk-assets
+```bash
+git clone https://github.com/supertuxkart/stk-code stk-code
+svn co https://svn.code.sf.net/p/supertuxkart/code/stk-assets stk-assets
 ```
 
-## Building on Linux
+## Building SuperTuxKart on Linux
 
 ### Dependencies
 
-Install the following packages:
+To build SuperTuxKart from source, you'll need to install the following packages:
 
   * OpenGL (mesa)
   * OpenAL (recommended: openal-soft-devel)
-  * Ogg    (libogg-dev)
+  * Ogg (libogg-dev)
   * Vorbis (libvorbis-dev)
   * Freetype (libfreetype6-dev)
   * libcurl (libcurl-devel)
   * libbluetooth (bluez-devel)
   * libpng (libpng-devel)
   * zlib (zlib-devel)
   * jpeg (libjpeg-turbo-devel)
+  * vpx (libvpx-devel)
```

Comment:
Benau's recording library may end up separate from STK, so maybe let's hold that one off a bit until it's clarifier how we handled video